<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Pismo</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="writingSpingBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Matematyka</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="mathSpinBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Inżynieria</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="engineeringSpinBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>Kalkulator punktów nauki (7 cudów świata)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="totalScoreLineEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zdobyte punkty</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="turnProgressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>8</number>
    </property>
    <property name="value">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="instructionLineEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>180</y>
      <width>481</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Zegar tur (Terraformacja Marsa)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continueInstructionButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Następne polecenie</string>
    </property>
   </widget>
   <widget class="QComboBox" name="extraChoice">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>271</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Zagraj kartę</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wykorzystaj projekt standardowy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Uzyskaj tytuł</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ufunduj nagrodę</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wykorzystaj akcję z karty aktywnej</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zamienienie roślinności w obszar zieleni</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zamienienie ciepła w Temperaturę</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="activateButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aktywuj</string>
    </property>
   </widget>
   <widget class="QListWidget" name="historyOfTurns">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>451</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="phasesLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Faza kolejności</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
